cmdstanpy/model.py

```diff
@@ -1343,17 +1393,18 @@ def _run_cmdstan(
                 if show_console:
                     lines = stdout.split('\n')
                     for line in lines:
-                        print('chain {}: {}'.format(idx + 1, line))
-
+                        print(f'{console_prefix}{line}')
             fd_out.close()
         except OSError as e:
             msg = 'Failed with error {}\n'.format(str(e))
             raise RuntimeError(msg) from e
         finally:
             fd_out.close()
 
-        if not show_progress:
-            get_logger().info('finish chain %d', idx + 1)
+        if show_console:
```

Comment:
I think we want to print the finished information even if show_console isn't true? Right now if you set both show_progress=False and show_console=False, do you not get any indication of the start/finish?

In general, most calls which use the logger should happen unconditionally. If a user doesn't want them, they can just assign `logger('cmdstan')` a `NullHandler`. The second call here should probably be debug, not info, though.

Reply:
> Right now if you set both show_progress=False and show_console=False, do you not get any indication of the start/finish?

per-chain logging messages mess up the TQDM progress bars in a bad way, so we don't want to do this in _run_cmdstan.
instead, added global "start processing / done processing" messages in the `sample` method.

reasoning: if you don't want to see the progress bars, then all you care about is overall completion.